intermediate_notebooks/E2E/taxi/NYCTaxi-E2E-GPU-Dask.ipynb

```diff
@@ -0,0 +1,1643 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
```

Comment:
Perhaps a markdown cell with some explanation of what this cell is for?

Perhaps your intent is to show that many of the features are of minimal importance and could be removed to save space in GPU mem?